canonical-web-and-design/vanilla-framework
# Add styling for detail element

## Done

Fixes #2455

## QA

- Pull code
- Run `./run serve --watch`
- Open http://0.0.0.0:8101/vanilla-framework/examples/detail
- Verify detail element has margin bottom
